model_bakery/baker.py

```diff
@@ -606,7 +606,17 @@ def _handle_one_to_many(self, instance: Model, attrs: Dict[str, Any]):
             manager = getattr(instance, key)
 
             for value in values:
-                if not value.pk:
+                # Django will handle any operation to persist nested non-persisted FK because
+                # save don't do so and, thus, raise constraint errors. That's why save() only gets
```

Comment:
Really minor grammatical change:

```suggestion
                # save doesn't do so and, thus, raises constraint errors. That's why save() only gets
```

Line may be too long after this change (can't tell from mobile).